questionnaire/questionnaire-service.js

```diff
@@ -107,26 +107,26 @@ function questionnaireService() {
         // dcs down...
         if (
             !result ||
-            !result.body ||
-            !result.body.data ||
-            !result.body.data.attributes ||
-            (result.body.errors && result.body.errors[0].status === 404)
+            !result.data ||
+            !result.data.data ||
+            !result.data.data.attributes ||
+            (result.data.errors && result.data.errors[0].status === 404)
         ) {
             const err = Error(`The service is currently unavailable`);
             err.name = 'DCSUnavailable';
-            err.statusCode = 500;
+            err.status = 500;
```

Comment:
Do we have a test in place for this, is it a valid integration test? 
Will need to test this within an integration environment.